core/src/display_object/edit_text.rs

```diff
@@ -1225,27 +1351,39 @@ impl<'gc> EditText<'gc> {
                         changed = true;
                     }
                 }
+                _ => {}
+            }
+            if changed {
+                let mut activation = Avm1Activation::from_nothing(
```

Comment:
I'm just doing exactly the same thing that's already done in the existing `text_input` function: https://github.com/ruffle-rs/ruffle/blob/1065662e845651bba173236b4a76b74d2881ce56/core/src/display_object/edit_text.rs#L1248-L1256

I don't know too much about this but I think if it's fine there it's fine here. And well, implementing whatever AVM2 feature/event may rely on this could be done in a separate PR, if that's indeed something that's missing (I have no idea)